addons/im_livechat/models/im_livechat_channel.py

```diff
@@ -162,6 +162,31 @@ def _get_livechat_mail_channel_vals(self, anonymous_name, operator=None, chatbot
             'public': 'private',
         }
 
+    def _get_livechat_session(self, anonymous_name, previous_operator_id=None, chatbot_script=None, user_id=None, country_id=None):
```

Comment:
looks too much duplicate, the only difference should be channel create vs return fake uuid (if possible, adapt prep commit accordingly)